static/src/javascripts/projects/common/modules/navigation/edition-picker.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'common/utils/fastdom-promise',
+    'common/utils/$'
+], function (
+    fastdomPromise,
+    $
+) {
+    var dropdown = $('.js-edition-picker-dropdown');
+
+    function enhanceToButton() {
+        var container = $('.js-edition-picker-container');
+        var checkBox = $('.js-enhance-checkbox');
+        var button = $.create('<button>');
+
+        button.addClass('edition-picker__button js-click');
+        button.attr('id', 'edition-picker');
+        button.attr('aria-controls', 'edition-picker__dropdown');
+        button.attr('aria-expanded', 'false');
+
+        fastdomPromise.write(function() {
+            checkBox.remove();
+            container.prepend(button);
+
+            openMenuOnClick();
+        });
+    }
+
+    function openMenuOnClick() {
+        var button = $('.js-click');
```

Comment:
Maybe this class could be more descriptive, or more certain to be unique to the `edition-picker`?